tools/github_readme_sync/hierarchy.py

```diff
@@ -68,13 +70,15 @@ def write_document(file, parent_slug, doc, indent_level):
 
 
 def check_hierarchy_file(folder: str):
+    folder = Path(folder)
     hierarchy = []
 
-    if not os.path.exists(os.path.join(folder, HIERARCHY_FILE)):
-        logging.error(f"File {os.path.join(folder, HIERARCHY_FILE)} does not exist")
+    hierarchy_file = folder / HIERARCHY_FILE
+    if not os.path.exists(hierarchy_file):
```

Comment:
I think this can be replaced with a Path method call.
```suggestion
    if not hierarchy_file.exists():
```